R/unnest-longer.R

```diff
@@ -0,0 +1,286 @@
+#' Unnest a list-column into rows
+#'
+#' @description
+#' `unnest_longer()` turns each element of a list-column into a row. It
+#' is most naturally suited to list-columns where the elements are unnamed
+#' and the length varies from row-to-row.
```

Comment:
```suggestion
#' and the length of each element varies from row-to-row.
```

I felt like this small bit of disambiguation helped with how I read this sentence